js/src/dataGrid/DataGridScope.ts

```diff
@@ -115,4 +115,10 @@ export class DataGridScope {
   setInitialSize() {
     this.dataGrid.setInitialSize();
   }
+
+  computeCSSVariablesAndRedraw() {
+    this.dataGrid.style = Theme.getStyle();
+    this.dataGrid.rowManager.createRows(this.store, this.store.selectHasIndex());
+    this.dataGrid.update();
```

Comment:
Maybe a call of `repaintBody` is enough here?

Reply:
Doesn't work, you need to create the rows again because it only reads the default color when creating the row.
Maybe go cell by cell changing the default colors...